stats/dashboard.py

```diff
@@ -1024,8 +1024,7 @@ def is_text_in_element(elementName):
             'contact-info': self.element.findall('contact-info/email'),
             'location': self.element.xpath('location/point/pos|location/name|location/description|location/location-administrative'),
             'location_point_pos': self.element.xpath('location/point/pos'),
-            'sector_dac': self.element.xpath('sector[@vocabulary="{}" or @vocabulary="{}" or not(@vocabulary)]'.format(self._dac_5_code(), self._dac_3_code())) if self._major_version() == '1'
-            else self.element.xpath('sector[@vocabulary="{}" or @vocabulary="{}" or not(@vocabulary)]'.format(self._dac_5_code(), self._dac_3_code())) or
+            'sector_dac': self.element.xpath('sector[@vocabulary="{}" or @vocabulary="{}" or not(@vocabulary)]'.format(self._dac_5_code(), self._dac_3_code())) or self._major_version() != '1' and 
```

Comment:
`[things] or self._major_version() != '1' and [other-things]`

`or` and `and` have the same precedence (source: https://www.tutorialspoint.com/python/operators_precedence_example.htm ). As such, there should be parentheses around `self._major_version() != '1' and [other-things]` to make it clearer how the statement is to be interpreted.